rust-runtime/smithy-eventstream/src/smithy.rs

```diff
@@ -32,12 +32,19 @@ expect_shape_fn!(fn expect_byte_array[ByteArray] -> Blob { bytes -> Blob::new(by
 expect_shape_fn!(fn expect_string[String] -> String { value -> value.as_str().into() });
 expect_shape_fn!(fn expect_timestamp[Timestamp] -> Instant { value -> *value });
 
+#[derive(Debug)]
 pub struct ResponseHeaders<'a> {
-    pub content_type: &'a StrBytes,
+    pub content_type: Option<&'a StrBytes>,
     pub message_type: &'a StrBytes,
     pub smithy_type: &'a StrBytes,
 }
 
+impl<'a> ResponseHeaders<'a> {
+    pub fn content_type_as_str(&self) -> Option<&str> {
```

Comment:
probably don't need `as_str` in the name here, I'd expect a content-type method to return a string

Reply:
Huh, I added that suffix since I thought the name would conflict with the `pub` member, but it seems to work.